app/models/case_management.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+##
+# Load the current CaseManagement client for the current environment.
+#
+module CaseManagement
+  def self.current
+    @current ||= "CaseManagement::#{config[:adapter]}".constantize.new
+  end
+
+  def self.current=(case_management)
+    @current = case_management
+  end
+
+  def self.config
+    @config ||= Rails.application.config_for(:case_management)
```

Comment:
Either need to handle file not existing or create a `config/case_management.yml` in CI. Favouring the latter option I think. Doesn't need to have correct details.

Reply:
Added in 4a65b3a just to make sure the specs actually pass with it. They do 🎉